packages/block-editor/src/components/inspector-controls/fill.js

```diff
@@ -19,9 +20,22 @@ import useDisplayBlockControls from '../use-display-block-controls';
 import groups from './groups';
 
 export default function InspectorControlsFill( {
-	__experimentalGroup: group = 'default',
 	children,
+	group = 'default',
+	__experimentalGroup,
 } ) {
+	if ( __experimentalGroup ) {
+		deprecated(
+			'`__experimentalGroup` property in `InspectorControlsFill`',
+			{
+				since: '6.2',
+				version: '6.4',
```

Comment:
Should we follow-up on these deprecations as we've reached the target version.

Reply:
PR is up removing the deprecations and old experimental prop here: https://github.com/WordPress/gutenberg/pull/56023